Mage.Sets/src/mage/cards/s/SpringLoadedSawblades.java

```diff
@@ -0,0 +1,63 @@
+package mage.cards.s;
+
+import mage.abilities.Ability;
+import mage.abilities.common.EntersBattlefieldTriggeredAbility;
+import mage.abilities.effects.common.DamageTargetEffect;
+import mage.abilities.keyword.CraftAbility;
+import mage.abilities.keyword.FlashAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.TargetController;
+import mage.filter.FilterCard;
+import mage.filter.FilterPermanent;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterArtifactCard;
+import mage.filter.common.FilterOpponentsCreaturePermanent;
+import mage.filter.predicate.permanent.TappedPredicate;
+import mage.target.TargetPermanent;
+
+import java.util.UUID;
+
+/**
+ * @author TheElk801
+ */
+public final class SpringLoadedSawblades extends CardImpl {
+
+    private static final FilterPermanent filter
+            = new FilterOpponentsCreaturePermanent("tapped creature an opponent controls");
+    private static final FilterCard filter2 = new FilterArtifactCard();
+
+    static {
+        filter.add(TappedPredicate.TAPPED);
+        filter2.add(TargetController.YOU.getOwnerPredicate());
+    }
+
+    public SpringLoadedSawblades(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ARTIFACT}, "{1}{W}");
+        this.secondSideCardClazz = mage.cards.b.BladewheelChariot.class;
+
+        // Flash
+        this.addAbility(FlashAbility.getInstance());
+
+        // When Spring-Loaded Sawblades enters the battlefield, it deals 5 damage to target tapped creature an opponent controls.
+        Ability ability = new EntersBattlefieldTriggeredAbility(new DamageTargetEffect(5, "it"));
```

Comment:
(fyi I've fixed most common triggered abilities to automatically replace "{this}" with "it" in rules text where applicable (still needed for activated abilities at the moment though))

Reply:
oh cool noted